### PR TITLE
Document decrypt function

### PR DESCRIPTION
**Pull request by sugar700**

### CODE REVIEW

File: encryption.py [hunk 1]
```diff
@@ -33,6 +33,20 @@ def create_decryptor(private_location, public_location):
...
+
+        Example:
+
+        >>> from cryptography.hazmat.backends import default_backend
+        >>> from cryptography.hazmat.primitives import hashes
+        >>> from cryptography.hazmat.primitives.asymmetric import padding
+        >>> from cryptography.hazmat.primitives.serialization import load_pem_public_key
+        >>> pkey = load_pem_public_key(open("server/pubkey.txt", "rb").read(), default_backend())
+        >>> message = b"Hello, world!"
+        >>> encrypted = pkey.encrypt(message, padding.OAEP(padding.MGF1(hashes.SHA1()), hashes.SHA1(), None))
```
Comment (phil-r) [inline encryption.py]: travis is not happy with this line 😞

**Comment by davidak** [comment]
@xfix what is your motivation for this change?

**Comment by sugar700** [comment]
@davidak: Just documenting that function, as it is good programming practice.

**Comment by davidak** [comment]
@xfix ah, i see. thought you add a function to decrypt a document.

**Comment by sugar700** [comment]
@davidak: This is essentially documenting usage of #227, contributed earlier. Essentially, it's to keep API keys secret, as they cannot be trusted to not be used to take over an account in a public repository. Public key to encrypt stuff can be downloaded from http://chaosthebot.com/pubkey.txt.

**Comment by phil-r** [comment]
@xfix Do you want to reopen it? I think it's important to have for #443 for example

**Comment by phil-r** [comment]
Just in case I've added documentation here: https://github.com/chaosbot/Chaos/wiki/Encryption

**Pull request closed without merging**